pkg/nb/types.go

```diff
@@ -414,6 +414,13 @@ type GenerateAccountKeysParams struct {
 	Email             string                `json:"email"`
 }
 
+// ResetPasswordParams is the params of account_api.reset_password()
+type ResetPasswordParams struct {
+	Email					string	`json:"email"`
+	VerificationPassword	string	`json:"verification_password"`
+	Password				string	`json:"password"`
```

Comment:
for the passwords use `MaskedString` instead of string. this way, when printing the params the passwords are replaces with `****`